processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryQueryToolChest.java

```diff
@@ -566,12 +566,24 @@ public Row apply(Object input)
                   DimensionHandlerUtils.convertObjectToType(results.next(), dimensionSpec.getOutputType())
               );
             }
-
             Iterator<AggregatorFactory> aggsIter = aggs.iterator();
+
+            // When using the result level cache, the agg values seen here are
```

Comment:
There's a long comment and identical block in each toolchest -- perhaps extract to a helper method in `CacheUtil`?

Reply:
Extracted to a helper method in `CacheStrategy` (`CacheUtil` is in `server` which is not a dependency of `processing`)